### PR TITLE
prettier fixes

### PR DESCRIPTION
**Pull request by sokra**

### CODE REVIEW

**Comment by sokra** [comment]
seeyoutomorrow @ijjk

**Review by ijjk — APPROVED** [review]
:seeya:

File: .prettierignore [hunk 1]
```diff
@@ -10,3 +10,6 @@ target/
 pnpm-lock.yaml
 crates/next-core/js/src/compiled
 crates/turbopack/bench.json
+
+# This file has intentional trailing commas
+crates/turbopack/tests/node-file-trace/integration/ts-package/tsconfig.json
```
Comment (jridgewell) [inline .prettierignore:15]: Should we exclude any `**/tests/**` directory from prettier?

**Review by jridgewell — APPROVED** [review]

**Review by mehulkar — APPROVED** [review]

File: .prettierignore (hunk 1 above)
Comment (ForsakenHarmony) [reply .prettierignore:15]: I don't think so

**Pull request merged**